custom/echis_reports/ucr/data_sources/household_characteristics_data_source.json

```diff
@@ -0,0 +1 @@
nt": null, "property_value": "yes"}]}, "expression_if_true": {"type": "constant", "constant": 1}, "type": "conditional", "expression_if_false": {"type": "constant", "constant": 0}}, "transform": {}, "is_primary_key": false, "is_nullable": true, "type": "expression", "column_id": "num_of_model_households"}, {"comment": null, "create_index": false, "display_name": "cbhi_enrolment", "datatype": "integer", "expression": {"test": {"type": "and", "filters": [{"operator": "eq", "type": "boolean_expression", "expression": {"expression": {"type": "coalesce", "expression": {"datatype": null, "type": "property_name", "property_name": "cbhi_member"}, "default_expression": {"type": "constant", "constant": "not_enroled_to_cbhi"}}, "type": "root_doc"}, "comment": null, "property_value": "yes"}, {"operator": "eq", "type": "boolean_expression", "expression": {"datatype": "string", "type": "property_name", "property_name": "type"}, "comment": null, "property_value": "family"}]}, "expression_if_true": {"type": "constant", "constant": 1}, "type": "conditional", "expression_if_false": {"type": "constant", "constant": 0}}, "transform": {}, "is_primary_key": false, "is_nullable": true, "type": "expression", "column_id": "cbhi_enrolment"}], "meta": {"doc_type": "DataSourceMeta", "build": {"doc_type": "DataSourceBuildInformation", "initiated": "2019-02-11T13:53:15.896670Z", "rebuilt_asynchronously": false, "app_id": null, "initiated_in_place": "2018-11-01T14:02:38.659276Z", "finished": true, "source_id": null, "app_version": null, "finished_in_place": true}}, "asynchronous": false, "base_item_expression": {}, "is_deactivated": false, "sql_settings": {"doc_type": "SQLSettings", "partition_config": []}, "configured_filter": {"type": "and", "filters": [{"operator": "eq", "type": "boolean_expression", "expression": {"datatype": null, "type": "property_name", "property_name": "closed"}, "comment": null, "property_value": false}, {"type": "or", "filters": [{"operator": "eq", "type": "boolean_expression", "expression": {"datatype": null, "type": "property_name", "property_name": "type"}, "comment": null, "property_value": "family"}, {"operator": "eq", "type": "boolean_expression", "expression": {"datatype": null, "type": "property_name", "property_name": "type"}, "comment": null, "property_value": "one_to_thirty"}, {"operator": "eq", "type": "boolean_expression", "expression": {"datatype": null, "type": "property_name", "property_name": "type"}, "comment": null, "property_value": "one_to_five"}]}]}, "backend_id": "SQL", "disable_destructive_rebuild": false, "description": "", "referenced_doc_type": "CommCareCase", "engine_id": "ucr", "last_modified": "2019-02-11T14:02:26.303716Z", "display_name": "(mobile) Household Characteristics Data Source", "named_expressions": {}, "doc_type": "DataSourceConfiguration", "icds_rebuild_related_docs": false, "table_id": "household_characterstics_tbl_id", "named_filters": {}, "_id": "02f1aecf16cae01da886e3fcea186f66", "sql_column_indexes": []}
```

Comment:
@andyasne You can use some online json formatter, to generate a readable version of the JSON (with correct spaces and widths). Can you update the two JSON files?

Reply:
I will do that and push again